apps/data/database/initdb.sql

```diff
@@ -97,6 +97,7 @@ CREATE TABLE user_clicks (
     click_id SERIAL PRIMARY KEY,
     user_id INTEGER NOT NULL REFERENCES users(user_id) ON DELETE CASCADE,
     listing_id INTEGER NOT NULL REFERENCES listings(listing_id),
+    click_term VARCHAR NOT NULL,
```

Comment:
Can we name this something a little more.....obvious? I had no clue what this was until I read the entire diff for this PR

Reply:
Done